context/app/api/vitessce_confs/base_confs.py

```diff
@@ -0,0 +1,176 @@
+import urllib
+from pathlib import Path
+import re
+
+from flask import current_app
+from vitessce import (
+    VitessceConfig,
+    MultiImageWrapper,
+    OmeTiffWrapper,
+    Component as cm,
+)
+
+from .utils import _get_matches
+from .constants import AssetPaths
+
+MOCK_URL = "https://example.com"
```

Comment:
This caused a circular import problem since the tests suite imports this file so moving this to the tests file would cause a circular import.  Maybe I can move this to the `constants.py` file or make a new one just for tests?